src/SFML/Audio/SoundFileReaderWav.cpp

```diff
@@ -288,6 +289,8 @@ bool SoundFileReaderWav::parseHeader(Info& info)
             // Store the starting position of samples in the file
```

Comment:
// Store the start and end position of samples in the file

... and remove that empty line between m_dataStart and m_dataEnd ;)